### PR TITLE
Feature/#5 BghController 추가

### LINKED ISSUE #5: BghController 추가
## Description
BghController 작성

### Controller
- [ ] BghController
## Additional context
백가현 controller 작성

### PR DESCRIPTION
**Pull request by GahBaek**
## Description
BghController 작성

### Controller
- [x] BghController
## Additional context
Closed #5

### CODE REVIEW

File: src/main/java/com/example/demo/controller/BghController.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.example.demo.controller;
+
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class BghController {
```
Comment (rnqhstmd) [inline src/main/java/com/example/demo/controller/BghController.java:6]: 괜찮네요

**Review by rnqhstmd — COMMENTED** [review]
머지해주세요

File: src/main/java/com/example/demo/controller/BghController.java (hunk 1 above)
Comment (jihyeonEom) [inline src/main/java/com/example/demo/controller/BghController.java:6]: 수고했어~~

**Review by jihyeonEom — COMMENTED** [review]
머지하자!

**Pull request merged**